pkgs/tools/X11/find-cursor/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub, libX11, libXdamage, libXrender, libXcomposite, libXext, installShellFiles, git }:
+
+stdenv.mkDerivation {
+  version = "1.6";
+  pname = "find-cursor";
+
+  src = fetchFromGitHub {
+    owner = "arp242";
+    repo = "find-cursor";
+    rev = "45e937a7a334373ee26b6e33341dfb79561d2767";
+    sha256 = "13lpcxklv9ayqapyk9pmwxkinhxah5hkr6n0jc2m5hm68nh220w1";
+  };
+
+  patches = [ ./man.patch ];
```

Comment:
A comment as to why this patch is required would be nice

Reply:
Thanks review.  I remove the unnecessary patch and prebuild the folder for man page.  :)